contracts/cw1-subkeys/src/contract.rs

```diff
@@ -1497,13 +1475,6 @@ mod tests {
         )
         .unwrap();
         assert_eq!(res.can_execute, true);
-        let res = query_can_execute(
```

Comment:
The new `Distribution::Withdraw` method should be supported in this high level interface. (People think of it as staking). Not sure how you want to divide the internal processing, but this API should remain similar